.eslintrc.json

```diff
@@ -0,0 +1,136 @@
+    "no-alert": "error",
+    "no-caller": "error",
+    "no-case-declarations": "error",
+    "no-eval": "error",
+    "no-extend-native": "error",
+    "no-extra-bind": "error",
+    "no-extra-label": "error",
+    "no-fallthrough": "error",
+    "no-floating-decimal": "error",
+    "no-implied-eval": "error",
+    "no-invalid-this": "error",
+    "no-iterator": "error",
+    "no-multi-str": "error",
+    "no-new-func": "error",
+    "no-new-wrappers": "error",
+    "no-new": "error",
+    "no-octal-escape": "error",
+    "no-octal": "error",
+    "no-proto": "error",
+    "no-redeclare": "error",
+    "no-return-assign": "error",
+    "no-script-url": "error",
+    "no-self-assign": "error",
+    "no-self-compare": "error",
+    "no-sequences": "error",
+    "no-throw-literal": "error",
+    "no-unmodified-loop-condition": "error",
+    "no-unused-expressions": "error",
+    "no-unused-labels": "error",
+    "no-useless-call": "error",
+    "no-useless-concat": "error",
+    "no-useless-escape": "error",
+    "no-void": "error",
+    "no-with": "error",
+    "radix": "error",
+    "wrap-iife": ["error", "inside"],
+
+    // Strict mode
+    "strict": ["error", "global"],
+
+    // Variables
+    "no-delete-var": "error",
+    "no-label-var": "error",
+    "no-restricted-globals": ["error", "event"],
+    "no-shadow-restricted-names": "error",
+    "no-undef-init": "error",
+    "no-undef": "error",
+    "no-unused-vars": ["error", { "vars": "local", "args": "none" }],
+
+    // Node.js
+    "handle-callback-err": "error",
+
+    // Stylistic issues
+    "array-bracket-spacing": ["error", "never"],
+//    "block-spacing": ["error", "always"], // TODO enable?
+    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+//    "comma-spacing": "error", // TODO enable?
+    "comma-style": ["error", "last"],
+    "eol-last": "error",
+//    "id-blacklist": ["error", "event"], // TODO enable?
+//    "indent": ["error", 2], // TODO enable?
+//    "key-spacing": ["error", { "beforeColon": false, "afterColon": true, "mode": "minimum" }], // TODO enable?
+    "keyword-spacing": "error",
+    "linebreak-style": ["error", "unix"],
+    "max-len": ["error", { "code": 200, "ignoreComments": true, "ignoreUrls": true }],
+    "new-cap": "error",
+    "new-parens": "error",
+    "no-array-constructor": "error",
+    "no-bitwise": "error",
+    "no-mixed-spaces-and-tabs": "error",
+    "no-multiple-empty-lines": ["error", { "max": 3, "maxEOF": 1 }],
+    "no-whitespace-before-property": "error",
+    "no-spaced-func": "error",
+    "no-trailing-spaces": "error",
+    "no-unneeded-ternary": "error",
+//    "object-curly-spacing": ["error", "never"], // TODO enable?
+//    "object-property-newline": ["error", { "allowMultiplePropertiesPerLine": true }], // TODO enable?
+//    "operator-linebreak": ["error", "after", { "overrides": { "?": "before", ":": "before" }}], // TODO enable?
+//    "quotes": ["error", "single"], // TODO enable?
```

Comment:
>
> > +// "quotes": ["error", "single"], // TODO enable?
>
> Should be enabled imo
>
I'd like to enable it, too, but not in this PR. Note that we currently
don't have this rule enabled (and we have lots of violation) so I don't
feel it has to be a part of the migration.